LOAN/LoansGeneral/LoansGeneral.rdf

```diff
@@ -464,9 +472,11 @@
 		</rdfs:subClassOf>
 		<rdfs:label>ownership interest</rdfs:label>
 		<skos:definition>classifier indicating the nature of the applicant&apos;s or borrower&apos;s ownership or leasehold interest in an asset used as collateral for the loan</skos:definition>
+		<fibo-fnd-utl-av:usageNote>Note that there are a number of variations for ownership interest that represent &apos;corner cases&apos;, including jurisdiction-specific variants, which can be added as needed for specific applications.</fibo-fnd-utl-av:usageNote>
```

Comment:
could use an example. 
How would it apply to the notion of Chattel in marine loans?